src/Orchard.Cms.Web/Startup.cs

```diff
@@ -41,6 +40,9 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseStaticFiles();
             loggerFactory.AddConsole(Configuration);
```

Comment:
I assume that here we have some configuration for the ConsoleLogger which is different from the NLog one? OR is it possible to used only NLog even for the console? And default the Console to Error, and keep Warning in the log file.